lib/cluster/cluster.c

```diff
@@ -162,6 +163,7 @@ pcmk_cluster_free(pcmk_cluster_t *cluster)
     if (cluster == NULL) {
         return;
     }
+    election_fini(cluster);
```

Comment:
`election_fini()` can be library-private now, if this is the only remaining caller

Reply:
thx